### DISCUSSION TITLE
Compatible with flask 3 ??

### DISCUSSION THREAD

**Discussion by gautierrog** [apache/airflow#42610] [Q&A] [answered]
Hello,

I want to use airflow as a library in my project and I realized airflow requires flask 2. Unfortunatly my application is using flask 3.

Is there a way to get a smaller set of airflow which does not require flask 2 in order to write my own dags ? If not, how would you recommand I integrate airflow with flask 3 ?

Cheers

**Comment by potiuk** [comment, marked as answer]
The dependency separation between different components is one of the the "features" of Airflow 3 (coming early next year). In Airflow 2 there is currently no way to do this separation.

**Reply by gautierrog** [reply]
Thanks for your answer. We look forward for Airflow 3 then :)
